noxfile.py

```diff
@@ -0,0 +1,173 @@
+######################################################################
+#
+# File: noxfile.py
+#
+# Copyright 2020 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+import os
+import subprocess
+
+import nox
+
+CI = os.environ.get('CI') is not None
+NOX_PYTHONS = os.environ.get('NOX_PYTHONS')
+
+PYTHON_VERSIONS = ['3.5', '3.6', '3.7', '3.8'] if NOX_PYTHONS is None else NOX_PYTHONS.split(',')
+PYTHON_DEFAULT_VERSION = PYTHON_VERSIONS[-1]
+
+PY_PATHS = ['b2', 'test', 'noxfile.py', 'setup.py']
+
+REQUIREMENTS_FORMAT = ['yapf==0.27']
+REQUIREMENTS_LINT = ['yapf==0.27', 'pyflakes', 'flake8==3.8.3', 'pytest==5.4.3']
```

Comment:
pyflakes not pinned?

Reply:
I thought I removed `flake8` for now and pinned `pyflakes`. Fixed